src/index.js

```diff
@@ -0,0 +1,30 @@
+import './style.css';
+import {
+  getStorageData, clearCompletedList, clearCheckBox, addTodo, saveEdit, displayToDo, getIsEditing,
+} from './module/addRemove.js';
+
+window.onload = () => {
```

Comment:
- [ ] Please consider changing `window.onload` to `window.addEventListener` 
      for better compatibility and best practice. :+1:

Reply:
I appreciate your advice and I have implemented it to enhance compatibility and best practice.
Thanks a lot :+1: